pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go

```diff
@@ -145,56 +146,70 @@ func (c *connectivityCheckController) Sync(ctx context.Context, syncContext fact
 		return err
 	}
 
+	existingChecks, err := c.operatorcontrolplaneClient.ControlplaneV1alpha1().PodNetworkConnectivityChecks(c.namespace).List(ctx, metav1.ListOptions{})
```

Comment:
no live calls outside of enabled.  If you need this outside enabled, then wire up a lister

Reply:
lister wire up is done now.

Reply:
@deads2k lister wireup is done properly now with a dependent PR in [cno](https://github.com/openshift/cluster-network-operator/pull/1649) and checks cleanup seems to be working fine. please have a look.